### PR TITLE
add initial working tensorflow model

### PR DESCRIPTION
**Pull request by davidcoh464**

### CODE REVIEW

File: src/fire_detector/fire_detector.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        Returns:
+            np.ndarray: The resized and padded image as a NumPy array.
+        """
+
+        # Calculate the aspect ratio
+        original_height, original_width, _ = image.shape
+        target_width, target_height = output_size
+        aspect_ratio = original_width / original_height
+
+        # Calculate the new dimensions while maintaining the aspect ratio
```
Comment (bezJac) [inline src/fire_detector/fire_detector.py]: remove redundant documentation inside function's , the code is self explanatory

File: src/fire_detector/fire_detector.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        """
+        Predict whether an image contains fire.
+
+        Args:
+            image (np.ndarray): The image to predict.
+
+        Returns:
+            bool: True if fire is detected, False otherwise.
+        """
+        desired_shape = (128, 128)
```
Comment (bezJac) [inline src/fire_detector/fire_detector.py]: use a constant

Comment (davidcoh464) [reply src/fire_detector/fire_detector.py]: done

File: src/fire_detector/fire_detector.py (hunk 1 above)
Comment (davidcoh464) [reply src/fire_detector/fire_detector.py]: done

**Review by bezJac — APPROVED** [review]
remove documentation from inside function logic. code is self explanatory

**Pull request merged**